### DISCUSSION TITLE
No ability to turn on clutterbar in both GTK and QT

### DISCUSSION THREAD

**Discussion by y0shir** [audacious-media-player/audacious#1381] [Q&A] [answered]
There is no option to disable hiding of "clutterbar"  in classic skin mode, which was possible in program that this mode tries to imitate.

**Comment by radioactiveman** [comment]
What do you mean with "clutterbar"? Please explain it with a screenshot.

**Reply by jlindgren90** [reply]
Clutterbar is the vertical row of capital letters O A I D V visible at the left of some classic Winamp skins. Essentially a tiny toolbar. O = Open Files, A = Add Files, D = Double Size, etc.

XMMS/Audacious probably supported it at some point in the past. I don't recall if we still have code for it currently or not.

**Reply by y0shir** [reply]
I'm pretty sure there is code, just that there is no option to turn it on.

**Comment by wildstar84** [comment, marked as answer]
I believe it's on (but hidden until one clicks or mouses down on it) in most skins (the "Default" skin being a notable exception, where it's turned off intentionally in skin.hints file).  If this is correct, then this is FIXED in Issue# 1513.

Regards,

Jim

**Reply by radioactiveman** [reply]
Yes, that's correct.

Starting with version 4.5 Audacious will show the clutterbar if it is not disabled explicitly in the skin.hints file with `mainwinMenurowVisible=0`.
